cmd/touch/touch_test.go

```diff
@@ -0,0 +1,61 @@
+package touch
```

Comment:
Nice tests :-)

Reply:
Thank you! Your review reminded me to add some tests when under filename is only directory name. I will split this one test function to smaller functions.